src/NextGen/WelcomeBanner/resources/views/welcome-banner.php

```diff
@@ -117,16 +132,16 @@
                             <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
                                 <path d="M10 4.166v11.667M4.165 9.999h11.667" stroke="#2D802F" stroke-width="1.667" stroke-linecap="round" stroke-linejoin="round"/>
                             </svg>
-                            <?php _e('Create a donation form using the new visual builder.', 'give'); ?>
+                            <?php _e('Create a next visual donation form', 'give'); ?>
```

Comment:
```suggestion
                            <?php _e('Create a new visual donation form', 'give'); ?>
```

Should this read `new` instead of `next`?

Reply:
@kjohnson I see. Yea well I personally like "Create a new visual donation form" more. I think it sounds more natural.

Reply:
@Benunc can you make a quick call on how this should read?

Reply:
@kjohnson @JoshuaHungDinh The _form_ is not "visual." The _builder_ is what's visual. So it should be the original "Create a form using the new visual builder"